src/containers/StructurePage/folderComponents/menuOptions/CopyResources.tsx

```diff
@@ -164,7 +164,7 @@ const CopyResources = ({
 
   const cloneResource = async (resource: Resource) => {
     const resourceType = resource.contentUri?.split(':')[1];
-    const resourceId = resource.contentUri?.split(':')[2] || '';
+    const resourceId = parseInt(resource.contentUri?.split(':')[2] || '');
```

Comment:
Er det ikke vanlig for ider å være strenger egentlig?

Reply:
ContentUri fra taksonomi kan i teorien komme på formatet urn:article:123#4 og då vil dette feile.

Reply:
Ikke fra det jeg har sett isåfall, har sjekka hele backenden og alle endepunktene som tar imot `id` er av typen `Long`.

Reply:
utenom det Gunnar sa ^

Reply:
Ah my bad, glemte å bytte om dette

Reply:
om du genererer en uuid så er det en streng, og vi behandler ikke egentlig ider som tall.

Reply:
Har vi noe uuid'er på contentUri? eller på articleId?

Reply:
vet ikke. tenkte bare på hva tilfelle er for en generell id løsning som brukes mange steder